dcos_test_utils/dcos_api.py

```diff
@@ -224,14 +225,20 @@ def login_default_user(self):
                 username or password of the default user.
         """
         if self.auth_user is None:
+            log.info('No credentials are defined')
+            return
+
+        if self.auth_user.auth_token is not None:
+            log.info('Already logged in as default user')
+            self.session.auth = DcosAuth(self.auth_user.auth_token)
             return
+
         log.info('Attempting default user login')
         # Explicitly request the default user authentication token by logging in.
         r = self.post('/acs/api/v1/auth/login', json=self.auth_user.credentials, auth=None)
         r.raise_for_status()
         log.info('Received authentication token: {}'.format(r.json()))
         self.auth_user.auth_token = r.json()['token']
-        self.auth_user.auth_cookie = r.cookies['dcos-acs-auth-cookie']
```

Comment:
Removing this may break some tests. This allows us to mimic the behavior of the browser using the web UI. e.g. https://github.com/dcos/dcos/blob/5a4e388ceabe1dc5769845ff458fb5b7694acbe3/packages/dcos-integration-test/extra/test_auth.py#L38-L44

So it appears that we always expect credentials to be used to login, and that the use of an authentication token is now broken.

Reply:
Ok. Nothing in the test utils seemed to use it.

Reply:
One test actually failed. I added the auth cookie again.